prices_voice.go

```diff
@@ -64,10 +76,10 @@ func (cvps *CountryVoicePriceService) Get(ctx context.Context, isoCountry string
 	return voicePrice, err
 }
 
-// returns the call price by number
-func (nvps *NumberVoicePriceService) Get(ctx context.Context, number string) (*VoiceNumberPrice, error) {
+// returns the call price by number or numbers pair
```

Comment:
Can we add a link to the API docs here? Apologies, as I should have done this the first time.

Reply:
done